snuba/web/db_query.py

```diff
@@ -341,6 +342,29 @@ def execute_query_with_rate_limits(
                 value=org_rate_limit_stats.rate,
             )
 
+        table_rate_limit_stats = rate_limit_stats_container.get_stats(
+            TABLE_RATE_LIMIT_NAME
+        )
+        if table_rate_limit_stats is not None:
+            metrics.gauge(
+                name="table_concurrent",
+                value=table_rate_limit_stats.concurrent,
+                tags={
+                    "cache_partition": reader.cache_partition_id
+                    if reader.cache_partition_id
+                    else "default"
+                },
+            )
+            metrics.gauge(
+                name="table_per_second",
+                value=table_rate_limit_stats.rate,
+                tags={
+                    "cache_partition": reader.cache_partition_id
+                    if reader.cache_partition_id
+                    else "default"
+                },
+            )
+
```

Comment:
this is fine but can we put it into its own function so this one doesn't get too large?

Reply:
Done.